forum/qa-plugin/user-activity-log/events-window/events-info-scripts.js

```diff
@@ -0,0 +1,15 @@
+const ModalWindow = ()=>{
+    const modal = document.querySelector('.modal-content');
+    const closeWindow = document.querySelector('.close-modal');
+    const openWindow = document.querySelector('.open-modal');
+
+    openWindow.addEventListener('click', ()=>{
+        modal.style.display = 'block';
+    }, false);
+
+    closeWindow.addEventListener('click', ()=>{
+        modal.style.display = 'none';
+    }, false);
+}
+
+window.onload = ModalWindow
```

Comment:
```suggestion
window.addEventListener('load', ...
```
pod `window.onload` jest już coś podpięte